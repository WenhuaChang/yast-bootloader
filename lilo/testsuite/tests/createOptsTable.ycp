/*
 * Authors:
 *  Unknown <unknown@addres.xx>
 *
 * $Id$
 *
 */

{
    include "testsuite.ycp";
    include "lilo/routines.ycp";

    TEST(``(createOptsTable($["opt1": "val1", "opt2": "val2", "image": "image"])), [], nil);

}
