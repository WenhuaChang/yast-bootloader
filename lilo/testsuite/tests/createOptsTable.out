Return	[`item (`id ("opt1"), "opt1", "val1"), `item (`id ("opt2"), "opt2", "val2")]
